plutarch-testlib/bench/Bench.hs

```diff
@@ -0,0 +1,25 @@
+module Main (main) where
+
+import Plutarch.LedgerApi.Utils (PMaybeData, pmaybeDataToMaybe, pmaybeToMaybeData)
+import Plutarch.Prelude
+import Plutarch.Test.Bench (BenchConfig (Optimizing), bcompare, bench, benchWithConfig, defaultMain)
+import Test.Tasty (testGroup)
+
+main :: IO ()
+main =
+  defaultMain $
+    testGroup
+      "Benchmarks"
+      [ testGroup
+          "pmaybeToMaybeData . pmaybeDataToMaybe"
+          [ bench
+              "non-optimized"
+              (plam (\m -> pmaybeToMaybeData #$ pmaybeDataToMaybe # m) # pconstant @(PMaybeData PInteger) (Just 42))
+          , bcompare
+              "$2 == \"pmaybeToMaybeData . pmaybeDataToMaybe\" && $3 == \"non-optimized\""
+              $ benchWithConfig
+                "optimized"
+                Optimizing
+                (plam (\m -> pmaybeToMaybeData #$ pmaybeDataToMaybe # m) # pconstant @(PMaybeData PInteger) (Just 42))
+          ]
```

Comment:
please also add examples using baseline here.

Reply:
It already supports that, baseline is done via CLI flags. You first generate csv file with `cabal run bench -- --csv baseline.csv` and then with no modifications to bench code you can do `cabal run bench -- --baseline baseline.csv`

Reply:
There's `cabal run bench -- --help` also

Reply:
Yeah, those are directly taken from how `tasty-bench` works.